.github/workflows/ci.yml

```diff
@@ -37,7 +37,29 @@ jobs:
       - name: Build package
         run: |
           python scripts/build/install.py
-
+      
+      - name: Build C++ Code with Coverage
+        run: |
+          mkdir -p build
+          cd build
+          cmake -DCMAKE_CXX_FLAGS="--coverage" ..
+          make
+  
+      - name: Run C++ Tests
+        run: |
+          cd build
+          ./test_binary  
+        # Capture and Generating Coverage Report with lcov
+      - name: Capture Coverage Data with lcov
+        run: |
+          cd build
+          lcov --capture --directory . --output-file coverage.info
```

Comment:
Add `lcov` in https://github.com/codezonediitj/pydatastructs/blob/main/environment.yml under `myst_nb`.

Reply:
working on this

Reply:
added lcov under myst_nb in enviromental.yml file